PSReadLine/BasicEditing.cs

```diff
@@ -115,17 +115,33 @@ public static void ForwardDeleteLine(ConsoleKeyInfo? key = null, object arg = nu
         }
 
         /// <summary>
-        /// Like BackwardKillLine - deletes text from the point to the start of the line,
+        /// Like BackwardKillLine - deletes text from the point to the start of the buffer,
+        /// but does not put the deleted text in the kill ring.
+        public static void BackwardDeleteBuffer(ConsoleKeyInfo? key = null, object arg = null)
```

Comment:
Maybe name it `BackwardDeleteInput`? `Buffer` is more of an implementation detail which may be confusing to the user.

Reply:
Agreed.